package.json

```diff
@@ -81,7 +81,7 @@
         "enzyme-to-json": "^3.3.1",
         "eslint": "^5.8.0",
         "eslint-plugin-react": "^7.11.1",
-        "gg-design-system": "^1.4.8",
+        "gumgum-design": "^1.9.20",
         "jest": "^24.1.0",
         "jest-cli": "^24.1.0",
         "node-sass": "^4.12.0",
```

Comment:
I think this should be removed in favor of sass. Trying to install the branch results in errors from `node-sass`.